lib/Config.js

```diff
@@ -847,47 +847,50 @@ class Config extends EventEmitter {
             this.utapi.localCache = config.localCache;
             assert(config.redis, 'missing required property of utapi ' +
                 'configuration: redis');
-            if (config.utapi.redis.sentinels) {
-                this.utapi.redis = { sentinels: [], name: null };
+            if (config.utapi.redis) {
```

Comment:
I'm not sure this is needed. I believe it was intended that the redis and utapi configs were separated. If the utapi config is defined, there also must be a redis config defined, of course (hence this assertion https://github.com/scality/cloudserver/pull/1934/files#diff-9280903abc69f09eff9ab36070b6d751R848). `redis` is just no longer a sub-object of `utapi`.

The redis configuration will be passed to Utapi here: https://github.com/scality/cloudserver/blob/development/8.0/lib/utapi/utilities.js#L9

Reply:
Although it looks like the config is implying as if utapi will be enabled without Redis, it's not. It's merely checking for redis existence as this particular test runs without enabling Utapi https://github.com/scality/cloudserver/blob/62df2c09c94ac1ab1d3bd48b4cd8c840e03f901d/lib/api/apiUtils/object/locationStorageCheck.js#L4.
Given that we are moving to using Utapi standalone in Zenko, I wouldn't worry too much about it.

Reply:
I don’t think Utapi will be enabled without Redis. I mean to point out that any values in `config.utapi.redis` will be overwritten by `config.redis` in any case.

Yes, Utapi will be standalone in Zenko—I think that’s another reason adding in `config.utapi.redis` isn’t needed.